docs/installation/Installing_ACME_Responder.md

```diff
@@ -0,0 +1,189 @@
+Installing ACME Responder
+=========================
+
+## Overview
+
+This document describes the process to install an ACME responder on a PKI server that already has a CA subsystem.
+It assumes that the CA was [installed](Installing_CA.md) with the default instance name (i.e. pki-tomcat).
+
+**Note:** The ACME responder is currently a tech preview which means:
+* It is not intended for production.
+* It may corrupt your data.
+* There is no guarantee for correctness, security, or performance.
+* There is no guarantee for documentation or support.
+* The API, configuration, or the database may change in the future.
+* There may be no easy upgrade path to the future version. 
+
+## Installing SANToCNDefault Policy
+
+The SANToCNDefault is a certificate profile policy which generates
+a default subject DN for the certificate in case the CSR does not provide one.
+The subject DN will be generated from the first DNS name in the the SAN extension.
+
+This policy is needed by the ACME profile, but currently it is not installed by default in the CA,
+so it has to be added manually.
+
+To add the policy, edit /etc/pki/pki-tomcat/ca/registry.cfg as follows:
+
+```
+defaultPolicy.ids=...,sanToCNDefaultImpl
+defaultPolicy.sanToCNDefaultImpl.class=com.netscape.cms.profile.def.SANToCNDefault
+defaultPolicy.sanToCNDefaultImpl.desc=SAN to CN Default
+defaultPolicy.sanToCNDefaultImpl.name=SAN to CN Default
+```
+
+**Note:** Restart the server to enable the policy.
+
+## Installing ACME Profile
+
+The acmeServerCert.cfg is a sample profile for generating server certificates via ACME responder.
+
+This profile is currently not installed by default in the CA, so it needs to be added and enabled manually.
+
+To add the profile, execute the following command:
+
+```
+$ pki -u caadmin -w Secret.123 ca-profile-add /usr/share/pki/ca/profiles/acmeServerCert.cfg --raw
+```
+
+To enable the profile, execute the following command:
+
+```
+$ pki -u caadmin -w Secret.123 ca-profile-enable acmeServerCert
+```
+
+## Installing ACME Responder
+
+To install the ACME responder on PKI server, execute the following command:
+
+```
+$ pki-server acme-create
+```
+
+The command will create the initial configuration files in /etc/pki/pki-tomcat/acme folder.
+
+## Configuring ACME Responder Database
+
+The database configuration for the ACME responder is located at /etc/pki/pki-tomcat/acme/database.json.
+
+To use an in-memory database, copy the sample configuration file with the following command:
+
+```
+$ cp /usr/share/pki/acme/conf/database/in-memory/database.json \
```

Comment:
@edewata I think with default setup InMemoryDatabase configuration would be there in /etc/pki/pki-tomcat/acme/database.json. So we can skip this step and we can highlight if customer wan't to add other custom  database config.

Reply:
@dpuniaredhat I'd caution against making `InMemoryDatabase` the default configuration.

 - It isn't persistent. Yes, this is in Tech Preview, but... Let's make people consider what they're doing and the implications of it. Not having persistent storage and a service restart wiping out every entry? I think that's unexpected _if_ `InMemoryDatabase` is the default.
 - There's no migration route between an `InMemoryDatabase` and `LDAPDatabase` or `PostgresDatabase` if/when we add those. 
 - We'd ideally display a warning if `InMemoryDatabase` is selected -- warning that persistence doesn't exist. Silently making it the default seems to go the other direction.

Keep in mind too, `InMemoryDatabase` is mostly for development and testing of this Tech Preview feature. When we get to a real release, we'll probably settle on a better, saner default. Those backends aren't yet written or not yet finalized yet.